ui/desktop/app/components/card/index.hbs

```diff
@@ -0,0 +1,15 @@
+<section class='credentials-card'>
```

Comment:
I'd use different terminology than `credentials`, since the card is more general.  Credentials just happens to be one (and currently only) use case.